aomi/helpers.py

```diff
@@ -215,9 +217,17 @@ def ensure_dir(path):
         os.mkdir(path)
 
 
+def clean_tmpdir(path):
+    """Invoked atexit, this removes our tmpdir"""
+    if os.path.exists(path) and \
+       os.path.isdir(path):
+        rmtree(path)
```

Comment:
Maybe for a future improvement but implementing something like `wipe`, `srm`, or `shred` to securely delete?

Reply:
Actually, that could be an :ok: extension to cryptorito... A subprocess wrapper around `srm` that is used on supported systems.